api/PayrollProcessor.Functions.Api/Features/Departments/DepartmentEmployeeTrigger.cs

```diff
@@ -1,6 +1,6 @@
 using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Logging;
-using Microsoft.WindowsAzure.Storage.Queue;
+using Microsoft.Azure.Storage.Queue;
```

Comment:
In the latest version of `Microsoft.Azure.WebJobs.Extensions.Storage` the namespaces of the Azure storage types used by the bindings has changed: https://github.com/Azure/azure-webjobs-sdk/releases/tag/storage-v4.0.0

Reply:
Good to know!